src/pardibaal/bound_t.cpp

```diff
@@ -67,15 +67,21 @@ namespace pardibaal {
     }
 
     bound_t bound_t::operator+(val_t rhs) const {
-        return bound_t(this->get_bound() + rhs, this->is_strict());
+        if (not this->is_inf())
```

Comment:
These three cases could deserve a test (for future reference)

Reply:
Good point